packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -25,6 +25,9 @@ const int _kDefaultSizeBytes = 100 << 20; // 100 MiB
 ///
 /// Generally this class is not used directly. The [ImageProvider] class and its
 /// subclasses automatically handle the caching of images.
+///
+/// A shared instance of this cache is retained by [PaintingBinding] and can be
+/// obtained via the [imageCache] top-level property.
```

Comment:
Should this be `[PaintingBinding.imageCache]` so it links directly there? As fas as I can tell, The ImageCache class does not have an imageCache property, right?

Reply:
It's a top-level property -- https://api.flutter.dev/flutter/painting/imageCache.html

It was actually linked in the docs before too, but the way it was structured, I missed the link.

Reply:
You are of course right, I think I was confused by the mentioning of PaintingBinding in the previous sentence and assumed you must mean the imageCache property on that thing.

Maybe you could clarify that [imageCache] is a top-level property in the painting library (https://master-api.flutter.dev/flutter/painting/painting-library.html). But it's also fine as is as the link will actually go to the right thing...